libethereum/ExtVM.cpp

```diff
@@ -33,7 +33,12 @@ namespace
 static unsigned const c_depthLimit = 1024;
 
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
-static size_t const c_singleExecutionStackSize = 12 * 1024;
+static size_t const c_singleExecutionStackSize =
+#ifdef NDEBUG
+	12 * 1024;
+#else
+	64 * 1024;
```

Comment:
I would like not to as we need it on Windons and OSX and testing on linux only. But I will try to improve it by reusing the same thread.